packages/xrpl/src/models/transactions/XChainModifyBridge.ts

```diff
@@ -0,0 +1,93 @@
+import { ValidationError } from '../../errors'
+import { Amount, XChainBridge } from '../common'
+
+import {
+  BaseTransaction,
+  GlobalFlags,
+  isAmount,
+  isXChainBridge,
+  validateBaseTransaction,
+} from './common'
+
+/**
+ * Enum representing values of {@link XChainModifyBridge} transaction flags.
+ *
+ * @category Transaction Flags
+ */
+export enum XChainModifyBridgeFlags {
+  /** Clears the MinAccountCreateAmount of the bridge. */
+  tfClearAccountCreateAmount = 0x00010000,
+}
+
+/**
+ * Map of flags to boolean values representing {@link XChainModifyBridge} transaction
+ * flags.
+ *
+ * @category Transaction Flags
+ */
+export interface XChainModifyBridgeFlagsInterface extends GlobalFlags {
+  /** Clears the MinAccountCreateAmount of the bridge. */
+  tfClearAccountCreateAmount?: boolean
+}
+
+/**
+ * The XChainModifyBridge transaction allows bridge managers to modify the parameters
+ * of the bridge.
+ *
+ * @category Transaction Models
+ */
+export interface XChainModifyBridge extends BaseTransaction {
+  TransactionType: 'XChainModifyBridge'
+
+  /**
+   * The bridge to modify.
+   */
+  XChainBridge: XChainBridge
+
+  /**
+   * The signature reward split between the witnesses for submitting attestations.
+   */
+  SignatureReward?: Amount
+
+  /**
+   * The minimum amount, in XRP, required for a {@link XChainAccountCreateCommit}
+   * transaction. If this is not present, the {@link XChainAccountCreateCommit}
+   * transaction will fail. This field can only be present on XRP-XRP bridges.
+   */
+  MinAccountCreateAmount?: Amount
+
+  Flags?: number | XChainModifyBridgeFlagsInterface
+}
+
+/**
+ * Verify the form and type of an XChainModifyBridge at runtime.
+ *
+ * @param tx - An XChainModifyBridge Transaction.
+ * @throws When the XChainModifyBridge is malformed.
+ */
+export function validateXChainModifyBridge(tx: Record<string, unknown>): void {
+  validateBaseTransaction(tx)
+
+  if (tx.XChainBridge == null) {
```

Comment:
A lot of this logic is shared between multiple transaction types.  Maybe have a base xchain validation function.

Reply:
This is already done with `isXChainBridge`. I don't think it matters that much for the two `if` statements that are replicated between all the transactions. I'm can change this if you object though.

Reply:
Fixed in https://github.com/XRPLF/xrpl.js/pull/2039/commits/6594c197ef14c3eded2784cd1437e89f416d4e9e - the helper functions aren't going to be used in existing models in this PR because that would be a lot of work, but the script should work for those as well.